app/src/main/java/org/p2p/wallet/sell/ui/payload/SellPayloadFragment.kt

```diff
@@ -111,6 +125,15 @@ class SellPayloadFragment :
         setButtonState(newState.cashOutButtonState)
     }
 
+    override fun updateToolbarTitle(tokenSymbol: String) {
```

Comment:
tokenSymbol is never empty, and it's always SOL

Reply:
always SOL now, not always SOL in the future